### DISCUSSION TITLE
Sidekiq for polling jobs.

### DISCUSSION THREAD

**Discussion by tomasmiguez** [sidekiq/sidekiq#6255] [Q&A]
We currently need to implement a polling job, and were thinking about using Sidekiq for this. Our idea was to raise an exception in case the external process was not done yet, and use Sidekiq's retry mechanism to keep polling. Would this be advisable, or is there any better mechanism available for that use case?
One issue that we have is that we use reporting tools that notify when an exception is raised. Is there any standard solution to stop that from being a problem? Or should we implement ad hoc ignoring logic for our particular use case? 

Thanks in advance!

**Comment by mperham** [comment]
I recommend using a minutely cron job to implement something recurring.
